templates/project/.github/workflows/test.yaml.twig

```diff
@@ -95,6 +95,12 @@ jobs:
         if: matrix.variant != 'normal'
         {% verbatim %}run: composer require ${{ matrix.variant }} --no-update{% endverbatim %}
 
+{# Remove when removing 3.x for the managed branches of block-bundle #}
+{% if repository_name == 'SonataBlockBundle' and '3.x' == current_branch %}
+      - name: Allow cyclic dependency between block-bundle and admin-bundle
+        run: sed --in-place "s/\"dev-master\":/\"dev-${GITHUB_SHA}\":/" composer.json
```

Comment:
This code was on travis, but for all bundles. IMO it is only needed on block bundle version 3: https://github.com/sonata-project/dev-kit/blob/master/templates/project/.travis/before_install_test.sh.twig#L22

Reply:
The dependency is only cyclic when developing the bundles, for the end users is not cyclic (is on require-dev of block-bundle and require of admin-bundle)